### DISCUSSION TITLE
Clarifying use of cleanStores vs "reseting" the store after tests

### DISCUSSION THREAD

**Discussion by marcebdev** [ai/nanostores#121] [Ideas]
I've noticed cleanStores doesn't "reset" the store to it's initial state after exiting a test, due to state living outside of React I think this could be a good thing. It seems to act more just like an `unmount` util to clean up after `keepMount()`.

Though I think the naming is confusing as I believed this would also dump the stores so the state would be "rebuilt" as a new copy on the next test.

I think it's worth first noting this behavior explicitly in the docs -- that it will not rebuild the store -- for clarity 

Then, I think it's worth considering either 2 functions. `unmount()` and `cleanStores()`, add a new sort of `dumpStores()` or something but that still leaves the naming of `cleanStores()` confusing, or make the first argument of `cleanStores()` take options. Obviously all have tradeoffs and there may be more desirable options.

FWIW, confusions aside, I think it's nice to have this functionality wrapped into a function without having to build your own `resetMyStore()` actions, which I do, but not every store warrants one. Especially when we're talking about Atoms and not Maps as having a reset action for every Atom probably doesn't serve any other functional purpose and would be very verbose.

**Comment by ai** [comment]
We can add value resetting if you know a simple way.
